client/FxAccountClient.js

```diff
@@ -1071,6 +1077,45 @@ define([
       });
   };
 
+  /**
+   * Send an unblock code
+   *
+   * @method sendUnblockCode
+   * @param {String} email email where to send the login authorization code
+   * @return {Promise} A promise that will be fulfilled with JSON `xhr.responseText` of the request
+   */
+  FxAccountClient.prototype.sendUnblockCode = function (email) {
+    required(email, 'email');
+
+    var data = {
+      email: email
+    };
+
+    var request = this.request;
```

Comment:
I just audited the rest of the file, everywhere there is a `var request = this.request`, `request` is used inside of a callback, so there was a real reason to set the reference. I had no such reason.